components/input/src/styles/notificationIcons.scss

```diff
@@ -57,6 +57,14 @@
   border: 0;
   background: unset;
   cursor: pointer;
+  outline-style: solid;
+  outline-width: 0;
+
+  &:focus-within,
+  &:focus:not(:focus-visible) {
+    // override wcss's core on flat button
+    outline-width: 1px;
```

Comment:
this code is not the solution we need. Do not release this as is. Remove this change and the a11y changes can be released. 

The new auro-button will have the correct styles for this and we should leverage that for any change like this. Also, see my comment on the ticket - the X icon was never intended to be focusable anyhow.

Reply:
None of these UI changes were reviewed with Design and should be considered blocked for release.